java/vector/src/main/java/org/apache/arrow/vector/BaseFixedWidthVector.java

```diff
@@ -42,7 +42,7 @@
  */
 public abstract class BaseFixedWidthVector extends BaseValueVector
         implements FixedWidthVector, FieldVector, VectorDefinitionSetter {
-  private final byte typeWidth;
+  private final int typeWidth;
```

Comment:
I have not changed the derived vector classes to use integer type. Should I?

Reply:
I don't think it matters.

Reply:
The `TYPE_WIDTH` const in the subclass is a static variable.

Reply:
for other types, they can be static, for fixed size binary, it has to be non-static

Reply:
Maybe if you can update `BitVector` since it currently casts down to a byte